docs/core/compatibility/sdk/6.0/duplicate-files-in-output.md

```diff
@@ -25,30 +25,22 @@ C:\Program Files\dotnet\sdk\5.0.100-preview.5.20258.6\Sdks\Microsoft.NET.Sdk\tar
 
 ## New behavior
 
-Starting in .NET 6, MSBuild removes duplicate files that are copied to the publish folder if both the source and destination are the same. If there are any remaining duplicates, a `NETSDK1148` error is generated and lists the files that are duplicated.
+Starting in .NET 6, MSBuild removes duplicate files that are copied to the publish folder if both the source and destination are the same. If there are any remaining duplicates, a `NETSDK1152` error is generated and lists the files that are duplicated.
 
 ## Reason for change
 
 Duplicate files in the publish output sometimes caused build breaks or unpredictable behavior.
 
 ## Recommended action
 
-Ideally, you should updated your project to avoid situations where multiple files with the same name are copied to the publish output.
+- Ideally, you should update your project to avoid situations where multiple files with the same name are copied to the publish output.
 
-Alternatively, you can set the [ErrorOnDuplicatePublishOutputFiles](../../../project-sdk/msbuild-props.md#erroronduplicatepublishoutputfiles) property to `false`.
+  For example, you may be unnecessarily copying a file to the output directory. Check the error message to find the name of the file that's being copied multiple times. Then, check the properties of that file to see if it's unnecessarily set to **Copy to Output Directory**.
```

Comment:
We can also mention here that [binlogs](https://github.com/dotnet/msbuild/blob/main/documentation/wiki/Providing-Binary-Logs.md) can be a good tool for debugging this kind of issue.